Formula/jack.rb

```diff
@@ -27,33 +16,22 @@ class Jack < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
+  depends_on "aften"
   depends_on "berkeley-db"
   depends_on "libsamplerate"
   depends_on "libsndfile"
-
-  on_linux do
-    depends_on "util-linux"
-  end
+  depends_on "python@3.9"
 
   def install
-    sdk = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
-
-    # Makefile hardcodes Carbon header location
-    inreplace Dir["drivers/coreaudio/Makefile.{am,in}"],
-      "/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h",
-      "#{sdk}/System/Library/Frameworks/Carbon.framework/Headers/Carbon.h"
-
-    # https://github.com/jackaudio/jack1/issues/81
-    inreplace "configure.ac", "-mmacosx-version-min=10.4",
-                              "-mmacosx-version-min=#{MacOS.version}"
-
-    system "./autogen.sh"
-    ENV["LINKFLAGS"] = ENV.ldflags
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "install"
+    # See https://github.com/jackaudio/jack2/issues/640#issuecomment-723022578
+    ENV.append "LDFLAGS", "-Wl,-compatibility_version,1" if MacOS.version <= :high_sierra
+    ENV.append "LDFLAGS", "-Wl,-current_version,#{version}" if MacOS.version <= :high_sierra
```

Comment:
Do we even need these lines anymore?

Reply:
You can keep them for people on older versions. It doesn't hurt.